src/test.jl

```diff
@@ -0,0 +1,612 @@
+
```

Comment:
Maybe we could call this file `ManifoldTests.jl`?

Reply:
Makes sense since it defines a separate module.